### PR TITLE
Interact with location nodes

### PR DESCRIPTION
**Pull request by matsoftware**
### Background

This PR enables the interaction with the direction boxes added to the scene.

### Breaking Changes

The `LNToucheDelegate` interfaces has been modified to notify the touch event of both an `AnnotationNode` and a `LocationNode`, that's represented by the parent of the `SCNBox` rendered.
The `PolylineNode` class is enriched with a `tag` attribute that can be used to identify the touched element.

### Meta
- Tied to Version Release(s):

### Checklist

- [ ] Appropriate label has been added to this PR (i.e., Bug, Enhancement, etc.).
- [ ] Documentation has been added to all `open`, and `public` scoped methods and properties.
- [ ] Changelog has been updated
- [ ] Tests have have been added to all new features. (not a requirement, but helpful)
- [ ] Image/GIFs have been added for all UI related changed.

### CODE REVIEW

**Comment by matsoftware** [comment]
(Closing the PR for now to do extra local tests)

**Comment by Pilot-Marc** [comment]
SCNnode already has a `name` property that appears to be designed for the very purpose of your `tag` property.

**Comment by matsoftware** [comment]
Hi @Pilot-Marc, I'm reusing the existing `tag` property of `LocationNode` for that. In the delegate method, to interact with the selected node, I'm doing something like this:
```swift
    func locationNodeTouched(node: LocationNode) {
        guard let name = node.tag else { return }
        guard let selectedNode = node.childNodes.first(where: { $0.geometry is SCNBox }) else { return }
...
```
The goal of this PR is to basically pass the identifier when generating multiple polylines.

**Comment by Pilot-Marc** [comment]
@matsoftware I was working on the same issue, detecting what node I was tapping on, but you beat me to it.

**Comment by matsoftware** [comment]
@aaronbrethorst / @Pilot-Marc Changelog and README updated

**Comment by intere** [comment]
@matsoftware - could you resolve the merge conflicts, please?

File: changelog.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
     - [PR #203 - InfoLabel includes additional heading data](https://github.com/ProjectDent/ARKit-CoreLocation/pull/203)
     - [PR #198 - Adds support for AR Orientation Tracking](https://github.com/ProjectDent/ARKit-CoreLocation/pull/198)
     - [PR #193 - Adds support for Swift Package Manager](https://github.com/ProjectDent/ARKit-CoreLocation/pull/193)
+    - [PR #197 - Interact with location nodes](https://github.com/ProjectDent/ARKit-CoreLocation/pull/197)
```
Comment (intere) [inline changelog.md]: Can you please move this line to the 1.2.2 section (1.2.1 has been released).

**Comment by matsoftware** [comment]
Changes pushed @intere

**Pull request merged**